dev/com.ibm.ws.jpa.container.core/resources/com/ibm/ws/jpa/jpa.nlsprops

```diff
@@ -302,6 +302,10 @@ NOT_COMPLIANT_WITH_JPA21_CWWJP0054E=CWWJP0054E: The {0} implementation, {1}, is
 NOT_COMPLIANT_WITH_JPA21_CWWJP0054E.explanation=The JPA persistence provider cannot be used because it lacks implementation of methods that are required by the JPA 2.1 specification.
 NOT_COMPLIANT_WITH_JPA21_CWWJP0054E.useraction=Obtain a newer version of the JPA persistence provider that complies with the JPA 2.1 specification.
 
+ERROR_TRANSFORMING_CLASS_CWWJP0055E=CWWJP0055E: An error occurred while attempting to transform the {0} Java Class.  The following failure detail has been gathered: {1}
```

Comment:
Change to
"An error occurred while attempting to transform the {0} Java class. The following failure detail was gathered: {1}"

Reply:
Great suggestion!  Implemented and updated.  Thanks!